fenics/environment.yml

```diff
@@ -0,0 +1,11 @@
+channels:
+- conda-forge
+dependencies:
+  - fenics =2019.1.0
+  - fenics-libdolfin =2019.1.0=*_8
+  - mshr =2019.1.0=*_2
+  - pyiron =0.3.21
+  - nglview =2.7.7
+  - matplotlib =3.3.3
+  - scikit-image =0.18.0
+  - seaborn =0.11.0
```

Comment:
Are these required? I thought `fenics` is only using `pyiron_base`. Or is it the image for Liam's lecture, in that case I would suggest to build it from `pyiron/md`.

Reply:
Thanks for your comment. I copied these from the environment.yaml which you used for the unit tests of the Liam's notebook. I am sure that Pyiron is required, but I am not sure about others. I will test them locally and remove the extra ones.

Reply:
the file, `fenics/environment.yml`, is now updated according to Liam's notebook.

Reply:
@jan-janssen, if there is no other issue with this pull request, shall I merge this branch?

Reply:
Please update the Readme and once merge the master branch into this one as we made quite a few changes to the master branch in the mean time.

Reply:
The changes in master are merged in this branch, the Readme is also updated.